app/utils/SubmissionFailureErrorCodes.scala

```diff
@@ -272,6 +299,9 @@ object SubmissionError {
     case ConsignorNotAuthorisedError.code => ConsignorNotAuthorisedError
     case RegisteredConsignorToRegisteredConsigneeError.code => RegisteredConsignorToRegisteredConsigneeError
     case ConsigneeRoleInvalidError.code => ConsigneeRoleInvalidError
+    case DispatchWarehouseExciseInvalid.code => DispatchWarehouseExciseInvalid
+    case DispatchWarehouseExciseInvalid2.code => DispatchWarehouseExciseInvalid2
+    case DispatchWarehouseExciseNotLinked.code => DispatchWarehouseExciseNotLinked
```

Comment:
Could you add some tests here: https://github.com/hmrc/emcs-tfe-create-movement-frontend/blob/f72004f744b6be87d86555412f1fd674650b53b1/test/utils/SubmissionFailureErrorCodesSpec.scala#L64